tests/bluetooth/tester/src/btp_mesh.c

```diff
@@ -1027,7 +1024,71 @@ static uint8_t proxy_solicit(const void *cmd, uint16_t cmd_len,
 static const struct bt_mesh_model root_models[] = {
 	BT_MESH_MODEL_CFG_SRV,
 	BT_MESH_MODEL_CFG_CLI(&cfg_cli),
+#if defined(CONFIG_BT_MESH_LARGE_COMP_DATA_SRV)
+	BT_MESH_MODEL_HEALTH_SRV(&health_srv, &health_pub, health_srv_meta),
+#else
 	BT_MESH_MODEL_HEALTH_SRV(&health_srv, &health_pub),
+#endif
```

Comment:
Won't this always work if `extern const struct bt_mesh_models_metadata_entry health_srv_meta[];` compiled without `ifdef` in `mesh/shell.h`? 
```suggestion
	BT_MESH_MODEL_HEALTH_SRV(&health_srv, &health_pub, health_srv_meta),
```